content/exchange/artifacts/Windows.Detection.DensityScout

```diff
@@ -0,0 +1,34 @@
+name: Windows.Detection.DensityScout
+description: |
+  Finding malware on a potentially infected system by calculating the density of each file under the C disk by default
+
+author: Raul Ramirez Gomez -- @rauramgom
+tools:
+  - name: densityscout
+    url: https://cert.at/en/downloads/software/software-densityscout
+    serve_locally: true
+
+precondition: SELECT OS From info() where OS = 'windows'
+
+required_permissions:
+  - EXECVE
+
+parameters:
+  - name: Arguments
+    description: |
+      A space separated list of args to run with.
+    default: 'C: -r -s exe,dll,sys,scr,ocx,cpl,pdf'
```

Comment:
try to make the args useful for a user - for example - what would a user need to tweak here?
maybe the list of extensions? Given just a blank command line is not as useful because the user needs to know how the tool works and what switches are available.

then have a parameter like "Extensions" which is comma separated and maybe takes a default value like this:

```yaml
parameters:
- name: Extensions
  default: exe,dll,sys,scr,ocx,cpl,pdf
```